web/docs/company/privacy.md

```diff
@@ -262,7 +262,7 @@ Some browsers also give you the option of downloading and installing extensions
-|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | We may use this information to localise features of the Service.                                                                                                        | The processing is necessary for our legitimate interest, namely localising features of the Service and tailoring the Service so that it is more relevant to our users.                                                         |                                                                                                                                                                                                                                                |
+|                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | We may use this information to localize features of the Service.                                                                                                        | The processing is necessary for our legitimate interest, namely localising features of the Service and tailoring the Service so that it is more relevant to our users.                                                         |                                                                                                                                                                                                                                                |
```

Comment:
Hey @Isaiah-Hamilton - all of this looks great except for this one change. 

We can't change our terms, even spelling, as it was drafted by the lawyers.

If you can undo this one change then I'll merge it in